internal/coreinternal/attraction/attraction.go

```diff
@@ -452,3 +452,17 @@ func getMatchingKeys(regexp *regexp.Regexp, attrs pcommon.Map) []string {
 	})
 	return keys
 }
+
+func Flatten(logger *zap.Logger, prefix string, src map[string]interface{}, dest map[string]interface{}) {
```

Comment:
why did we choose this file? I can see that it already has some purpose which seemed like some CRUD operation.
I see that better place would have been under:
- Either, in a new file `internal/common/sanitize/attributes.go`
- Or, in `internal/common/maps/maps.go`